src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -374,11 +382,19 @@ public AgentSettings LoadSettings()
 #endif
                 }
 
-                //delete agent from the server
```

Comment:
1. I want to stop the listener process first before the agent gets removed from the server. 
2. IsServiceConfigured() is dependent on config store and it gets cleaned up.

---
In reply to: [117651647](https://github.com/Microsoft/vsts-agent/pull/978#discussion_r117651647) [](ancestors = 117651647)